### DISCUSSION TITLE
How to Save Video and Image Automatically

### DISCUSSION THREAD

**Discussion by james9078** [sofa-framework/sofa#3730] [Develop in C++] [answered]
Hi,

With SofaPython3, is there a way to automatically start the animation and save the entire simulation video and the image from the last time step? 
I am trying to run simulations at scale by deploying SOFA on cloud which means we need to be able to run simulations from only the command line. 

Thank you.

**Comment by fredroy** [comment, marked as answer]
Hello,
If you are deploying SOFA on cloud, it means you wont have access to any GUI right ?
If so, you may have a look at https://github.com/jnbrunet/SofaOffscreenCamera , which should do what you want.
Be aware that it should not be maintained by its original author anymore (or at least he wont answer quickly)
